tests/test_dataset.py

```diff
@@ -857,6 +857,39 @@ def test_features_transform_logic_graphdataset(self):
         assert dataset_train.means == dataset_test.means
         assert dataset_train.devs == dataset_test.devs
 
+    def test_inherit_info_training(self):
```

Comment:
I'd test two cases: the first in which I assign node_features, edge_features, features_transform, target and task in the training dataset only, and I verify that if I define a test dataset flagging train as False and passing in dataset_train the attributed are the same; then a second case as you did below, in which I define a training set and then I re-assign parameters in the test set, and I verify that they are still the same as in the training set.